docs/reference-guides/monitoring-v2-configuration/receivers.md

```diff
@@ -211,6 +213,87 @@ url http://rancher-alerting-drivers-sachet.ns-1.svc:9876/alert
+
+#### Create an AlertManagerConfig
+
+1. Click on **Monitoring**; then click on **Alerting**, and open **AlertManagerConfigs**.
+2. Click on **Create**
+
+#### Create a Receiver in AlertManagerConfig
+
+1. Choose a namespace from the dropdown and set an appropriate name. Click **Create**.
+2. Open the newly created **AlertManagerConfig**. Look for three dots near the top-right of the page and click **Edit Config**.
+3. Create a new Receiver by clicking **Add Receiver**.
+4. Select **Webhook** from the list that shows up on the page **Create Receiver in AlertmanagerConfig**. 
+5. Name the webhook. Then click **Add Webhook**. In the **Select Webhook Type** dropdown select **SMS**. This will automatically populate the **Target** field as `http://rancher-alerting-drivers-sachet.cattle-monitoring-system.svc:9876/alert`. If you installed the **Alerting Drivers** in a namespace other than `cattle-monitoring-system`, above URL will be reflect that. 
+6. Next click **Create** near the bottom of the page.
+
+#### Create a Route in AlertManagerConfig
+
+1. After creating a **Receiver** once again look for the three dots near the top-right and click **Edit Config**.
+2. Click on **Route**.
+3. In the dropdown, select the **Receiver** we just created in above step
+4. In the **Labels to Group Alerts By** field, type `test`. We set this group name while creating **PrometheusRule** above.
+5. Under the **Waiting and Intervals** set **Group Wait** value to `1s` and **Group Interval** value to `10s`. This will cause frequent alerts to be sent, if configured correctly. Change the values as appropriate.
+6. Under **Matchers** click on **Add Matcher** and enter the **Name** as `test` and **Value** as `alert`. We used these values as **Label** in the **PrometheusRule** we created earlier. From the **Match Type** dropdown, select `MatchEqual`.
+7. Click **Save**.
+
+### Configuring the Telegram Receiver
+
+The Telegram receiver can be configured by updating the `rancher-alerting-drivers-sachet` ConfigMap in `cattle-monitoring-system` namespace. For example, the following is a minimal Telegram receiver configuration.
+
+```yaml
+providers:
+  telegram:
+    token: <your-token-from-telegram>
+
+receivers:
+- name: 'cattle-monitoring-system/test-amc/prom2tel'
+  provider: 'telegram'
+  to:
+    - '123456789'
+```
+
+To obtain a Telegram token, you will have to setup a Telegram bot. Refer [this guide](https://core.telegram.org/bots/tutorial) for details.
+When configuration is complete, add the receiver using the steps in [this section](#creating-receivers-in-the-rancher-ui). 
+
+Use the `<namespace>/<alertmanagerconfig-name>/<receiver-name>` as the name of the receiver above. The numeric `123456789` is the Telegram user ID you would like to send the notifications to. To find this Telegram ID, check out [this Telegram bot](https://telegram.me/userinfobot).
```

Comment:
```suggestion
Name the receiver `<namespace>/<alertmanagerconfig-name>/<receiver-name>`. Enter `123456789` as the Telegram user ID to send the notifications to. To find this Telegram ID, check [the Telegram userinfo bot](https://telegram.me/userinfobot).
```
Is `12345678` just a placeholder? Why do they need to check the bot to find the ID if we told them which ID to use?

Reply:
> Is 12345678 just a placeholder?

Yes, it is.

> Why do they need to check the bot to find the ID if we told them which ID to use?

Because that's how the numeric user ID for your Telegram username can be found. I don't know of another way myself. It's also the one mentioned in [sachet example](https://github.com/messagebird/sachet/blob/master/examples/config.yaml#L121-L124).